addons/project/views/project_update_templates.xml

```diff
@@ -19,8 +19,7 @@
 <h1 style="font-weight: bolder;">Activities</h1>
 </div>
 
-<div name="milestone">
-<t t-if="milestones['show_section']">
```

Comment:
I think we need to keep this as otherwise it could break in custos where some might have made an x-path on the t-if with a position="inside" in order to append some content in it.

Reply:
Fixed, thanks for the review